### PR TITLE
Playground initial, Indented formatting

### LINKED ISSUE #80: Playground- Default code contents
Default code contents on initial Playground load (if not following a direct link with existing contents)

What should code contents be?

### PR DESCRIPTION
**Pull request by jamesnw**
## Description
Adds an initial Sass document, showing modules, lists, variables, and nesting. 

This also adds Formatting to the Indented mode, previously in #79 

## Related Issue(s)
#80

## Steps to test/reproduce
From homepage, click on Playground, and see there is content. Switch to Sass indented format, and see there is initial content in the indented format.

Make any changes, and reload, and see that your changes persist.

https://deploy-preview-81--sass-site-oddbird.netlify.app/playground

- [x] @jgerigmeyer  Review code
- [x] @mirisuzanne  Review default content- is it a good example?

### CODE REVIEW

File: source/assets/js/playground.ts [hunk 1]
```diff
@@ -20,20 +26,32 @@ function setupPlayground() {
   const hash = location.hash.slice(1);
   const hashState = deserializeState(hash);
 
+  const inputFormat = hashState.inputFormat || 'scss';
+
   const initialState: PlaygroundState = {
-    inputFormat: hashState.inputFormat || 'scss',
+    inputFormat: inputFormat,
```
Comment (jgerigmeyer) [inline source/assets/js/playground.ts]:
```suggestion
    inputFormat,
```

**Review by jgerigmeyer — APPROVED** [review]
Works great, code looks good! 🚀

File: source/assets/js/playground/editor-setup.ts [hunk 2]
```diff
@@ -77,4 +95,40 @@ const outputSetup = (() => [
...
+}
+
+a {
+  color: $primary-color;
+  &:hover{
+    color: color.adjust($primary-color, $lightness: 20%);
+  }
+}
+
+@debug list.append($font-stack, Arial);`,
```
Comment (mirisuzanne) [inline source/assets/js/playground/editor-setup.ts]:
```suggestion
  indented: `@use "sass:list";
@use "sass:color";

$font-stack: Helvetica, Arial;
$primary-color: #333;

body {
  $font-stack: list.append($font-stack, sans-serif);
  font: $font-stack;
}

a {
  color: $primary-color;

  &:hover{
    color: color.scale($primary-color, $lightness: 20%);
  }
}

@debug $font-stack;`,
```

**Review by mirisuzanne — APPROVED** [review]

**Comment by jamesnw** [comment]
Moved to sass repo - https://github.com/sass/sass-site/pull/1164

**Pull request closed without merging**